scripts/setup_database.py

```
@@ -12,6 +12,12 @@
 sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
 from config.settings import DATABASE_URL, settings
+from models.database_models import (
+    ChatbotSession, ChatMessage, ChatbotIssue,
```

Ruff (0.12.2)

`models.database_models.ChatbotSession` imported but unused

Remove unused import

(F401)